src/components/ApplicantOnboardingForm/ApplicantProfileForm.js

```diff
@@ -28,9 +28,15 @@ export default function ApplicantProfileForm() {
   const dispatch = useDispatch();
   const classes = useStyles();
   const userId = useSelector((state) => state.login.user.id);
-  // active step keeps track of which child component will render
+
+  /* ********************* BEGIN FORM STATE AND SETTERS ********************* */
```

Comment:
These comments are really awesome. Super helpful for our team to understand what is going on and for future devs that will work on this app.